public/video-ui/src/routes.tsx

```diff
@@ -29,7 +30,7 @@ export const routes = (
       <Redirect from="/videos/create" to="/create" />
       <Route path="/videos/:id" component={Video} onEnter={sendTelemetry} />
       <Route path="/videos/:id/upload" component={Upload} onEnter={sendTelemetry} />
-      <Route path="/create" component={Video} mode="create" onEnter={sendTelemetry} />
+      <Route path="/create" component={Video} props={{mode: "create"}} onEnter={sendTelemetry} />
```

Comment:
This seems to be the way ReactRouter 3 expects to pass props to components, however the previous version was valid, just resulting in type errors.